src/Workspaces/Core/Portable/Workspace/Solution/Project.cs

```diff
@@ -35,55 +35,31 @@ internal Project(Solution solution, ProjectState projectState)
             _projectState = projectState;
         }
 
-        internal ProjectState State
-        {
-            get { return _projectState; }
-        }
+        /// <summary>
+        /// True if project information is complete. <see cref="ProjectInfo.IsComplete"/> for more info
+        /// </summary>
+        internal bool IsComplete => _projectState.ProjectInfo.IsComplete;
```

Comment:
same reason as HasCompleteReferenceAsync is internal. probably we need to make both public, but right now it is internal.